Library/Homebrew/utils/bottles.rb

```diff
@@ -57,6 +57,7 @@ def resolve_version(bottle_file)
     class Bintray
       def self.package(formula_name)
         formula_name.to_s.tr("+", "x")
+        formula_name.to_s.sub!(/(.)@(\d)/, "\\1AT\\2") # Handle foo@1.2 style formulae.
```

Comment:
Thanks. I did actually test this a bit, but I managed to confuse myself by doing:

```ruby
fname = Formula["wget"].name
fname.to_s.sub!(/(.)@(\d)/, "\\1AT\\2")
=> nil
fname
=> "wget"
```
I managed to miss the `fname` part wouldn't be automatically triggered.